lib/engine/game/g_1822_pnw/step/merge.rb

```diff
@@ -46,7 +46,14 @@ def buyer
           def mergeable(corporation)
             return [] unless @merge_state == :none
 
-            @game.unassociated_minors.select do |m|
+            mergeable_minors =
+              if @game.associated_minor?(corporation)
+                @game.unassociated_minors
+              else
+                @game.associated_minors
+              end
```

Comment:
Nit: this could be condensed to `@game.associated_minor?(corporation) ? @game.unassociated_minors : @game.associated_minors`